aspnet/tutorials/your-first-mac-aspnet.rst

```diff
@@ -1,40 +1,62 @@
 Your First ASP.NET 5 Application on a Mac
 =========================================
 
-By `Steve Smith`_
+By `Steve Smith`_, `Daniel Roth`_
 
 ASP.NET 5 is cross-platform and can be developed and run on Mac OS X as well as Linux and Windows. See how you can quickly install, scaffold, run, debug, and deploy ASP.NET applications on a Mac.
 
 In this article:
-	- `Setting Up Your Development Environment`_
-	- `Scaffolding Applications Using Yeoman`_
-	- `Developing ASP.NET Applications on a Mac With Visual Studio Code`_
-	- `Running Locally Using Kestrel`_
-	- `Publishing to Azure`_
+  - `Setting Up Your Development Environment`_
+  - `Scaffolding Applications Using Yeoman`_
+  - `Developing ASP.NET Applications on a Mac With Visual Studio Code`_
+  - `Running Locally Using Kestrel`_
+  - `Publishing to Azure`_
 
 Setting Up Your Development Environment
 ---------------------------------------
 
-First, make sure you have :doc:`installed ASP.NET on your Mac OS X machine </getting-started/installing-on-mac>`. This step will include installing `Homebrew <http://brew.sh/>`_ and configuring DNVM. This article has been tested with DNX beta4. You can confirm that you are running the correct version of ``dnx`` by running the command ``dnvm list``. You should see ``default`` listed under the ``Alias`` column for ``1.0.0-beta4`` as shown:
-
-.. image:: your-first-mac-aspnet/_static/dnvm-list-beta4.png
-
+First, make sure you have :doc:`installed ASP.NET on your Mac OS X machine </getting-started/installing-on-mac>`. This step will include installing the .NET Execution Environment (DNX) and the .NET Version Manager (DNVM). You can check which DNX version you have active by running ``dnvm list``.
```

Comment:
In this case I wanted to control the friendly text that gets displayed for the link